src/Graphics/Renderer.h

```diff
@@ -75,16 +78,17 @@ namespace Apex {
 		void Push();
 		void Pop();
 
-		void Drawtext(const Vec2& position, const std::string& asciiText, float quadHeight, Font* font, Shader shader);
-		void DrawQuad(const Vec2& position, const Vec2& dimensions, const Texture& texture, const AABB2& texCoords, const Vec4& color, Shader shader);
-		void DrawQuad(Mesh* mesh, Shader shader);
-		void DrawFrameBuffer(Mesh* mesh);
+		void DrawMesh(const Vec2& position, const std::string& asciiText, float quadHeight, Font* font, Shader shader);
```

Comment:
These will still be called DrawText, DrawQuad.. Just internally they  will call DrawMesh after setting up whatever they need to

Reply:
Also why is this change even part of this branch when you have a separate branch to do this refactor?

Reply:
This was done by mistake bhaiya I have rectified it in my new branch "RendererRefactor" when I merge this branch the whole mistake will be corrected